internal/commands/gateways/command.go

```diff
@@ -0,0 +1,40 @@
+package gateways
+
+import (
+	"github.com/mitchellh/cli"
+)
+
+func New() *Command {
+	return &Command{}
+}
+
+type Command struct{}
+
+func (c *Command) Run(args []string) int {
+	return cli.RunResultHelp
+}
+
+func (c *Command) Synopsis() string {
+	return synopsis
+}
+
+func (c *Command) Help() string {
+	return help
+}
+
+const synopsis = "Manage Consul API Gateways"
+const help = `
+Usage: consul-api-gateway gateways <subcommand> [options] [args]
+  This command has subcommands for interacting with Consul API Gateways
+  configuration objects. Here are some simple examples, and more detailed examples are
+  available in the subcommands or the documentation.
```

Comment:
Are we able to add a link for the documentation?  Knowing that this is internal might not be as important to include, but could be helpful either way.

Reply:
Yeah, that's totally possible, this would just be a command line usage string for folks using the CLI, so nothing really "internal" about it. I'm imagining that we'd land somewhere like https://www.consul.io/docs/api-gateway (or maybe a subpage with runtime-specific docs?).

Any preference on exactly what we link here?

Reply:
Yea I think adding a link to a subpage within docs that has relevant content based on where the user is would be great! However, there are a few places where you reference linking out to documentation (I just called out the two, but happy to comment on the others for visibility if you want) if it is easier to provide a single link for all references than linking to the overview page would also be helpful to get them to a starting point.

Reply:
Ok, sure thing, for now I can just add the top-level documentation links for all the subcommands we're adding. I'm sure as we get closer to shipping this we'll be adding some subpages to the docs that we can link out to more particularly.

Thanks for the quick pass on the copy and suggestions. I'm sure we'll want your pass when we actually implement the output of the commands as well, so I'll tag you on that when we get there too!